src/file.coffee

```diff
@@ -169,6 +184,14 @@ class File
         @realPath = @path
     @realPath
 
+  # Public: Returns a promise that resolves to the file's completely resolved {String} path.
+  getRealPath: ->
+    if @realPath?
+      Q.resolve @realPath
```

Comment:
Are `Q.resolve(@realPath)` and `Q(@realPath)` functionally the same thing?

If so, maybe this should be `Q(@realPath)` to be consistent with other Q usage in this file.